tests/Core/Location/Geolocation.cpp

```
@@ -0,0 +1,22 @@
+
+class GeolocationTest : public ::testing::Test {
+protected:
+  const double latitude = 35.689487;
+  const double longitude = 139.691711;
+  Geolocation location{latitude, longitude};
+};
+
+TEST_F(GeolocationTest, GeolocationPlusCodeTest) {
+  auto result = location.GetPlusCode();
+  EXPECT_NE(result, "Not Valid");
+}
```

Cppcheck (2.10-2)

[performance] Variable 'm_StartPoint' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)